packages/board-utils/src/board-repr.ts

```diff
@@ -1,11 +1,32 @@
-import { schema } from "@sudoku-studio/schema";
+import { schema, ArrayObj, IdxBitset, Geometry } from "@sudoku-studio/schema";
 
 export function makeUid(): string {
     return `${(31 * Math.floor(0xFFFFFFFF * Math.random()) + Date.now()) % 0xFFFFFFFF}`;
 }
 
 export type CreateElementFn = <E extends schema.Element>(type: E['type'], value?: E['value']) => E;
 
+function defaultRegions(size: number = 9, boxWidth: number = 3, boxHeight: number = 3): ArrayObj<IdxBitset<Geometry.CELL>> {
+
+    const regions: ArrayObj<IdxBitset<Geometry.CELL>> = {};
+    for (let i = 0; i < size; i++) {
+        regions[i] = {};
+    }
+
+    for (let y = 0; y < size; y++) {
+        for (let x = 0; x < size; x++) {
+            // Inlined to prevent circular dependency.
+            // const grid: schema.Grid = { width: size, height: size };
+            // const cellIdx = cellCoord2CellIdx([ x, y ], grid);
```

Comment:
Can you remove this? (I think we generally prefer to not have commented out code)